entrypoint.bash

```diff
@@ -2,42 +2,337 @@
+            mkdir -p android/kernel/coral
+            cd android/kernel/coral
+            repo init -u https://github.com/GrapheneOS/kernel_manifest-coral.git -b 13
+            repo sync -j${NPROC_SYNC}
+            KBUILD_BUILD_VERSION=1 KBUILD_BUILD_USER=build-user KBUILD_BUILD_HOST=build-host KBUILD_BUILD_TIMESTAMP="Thu 01 Jan 1970 12:00:00 AM UTC" BUILD_CONFIG=private/msm-google/build.config.sunfish build/build.sh
+            rsync -av --delete out/android-msm-pixel-4.14/dist/ device/google/sunfish-kernel/
+            ;;
+        bramble|redfin|barbet)
+            mkdir -p android/kernel/redbull
+            cd android/kernel/redbull
+            repo init -u https://github.com/GrapheneOS/kernel_manifest-redbull.git -b 13
+            repo sync -j${NPROC_SYNC}
+            BUILD_CONFIG=private/msm-google/build.config.redbull.vintf build/build.sh
+            rsync -av --delete out/android-msm-pixel-4.19/dist/ device/google/redbull-kernel/vintf/
+            ;;
+        oriole|raven)
+            mkdir -p android/kernel/raviole
+            cd android/kernel/raviole
+            repo init -u https://github.com/GrapheneOS/kernel_manifest-raviole.git -b 13
+            repo sync -j${NPROC_SYNC}
+            LTO=full BUILD_AOSP_KERNEL=1 ./build_slider.sh
+            rsync -av --delete out/mixed/dist/ device/google/raviole-kernel/
+            ;;
+        bluejay)
+            mkdir -p android/kernel/bluejay
+            cd android/kernel/bluejay
+            repo init -u https://github.com/GrapheneOS/kernel_manifest-bluejay.git -b 13
+            repo sync -j${NPROC_SYNC}
+            LTO=full BUILD_AOSP_KERNEL=1 ./build_bluejay.sh
+            rsync -av --delete out/mixed/dist/ device/google/bluejay-kernel/
+            ;;
+        panther|cheetah)
+            mkdir -p android/kernel/pantah
+            cd android/kernel/pantah
+            repo init -u https://github.com/GrapheneOS/kernel_manifest-pantah.git -b 13
+            repo sync -j${NPROC_SYNC}
+            LTO=full BUILD_AOSP_KERNEL=1 ./build_cloudripper.sh
+            rsync -av --delete out/mixed/dist/ device/google/pantah-kernel/
+            ;;
+        lynx)
+            mkdir -p android/kernel/lynx
+            cd android/kernel/lynx
+            repo init -u https://github.com/GrapheneOS/kernel_manifest-lynx.git -b 13
+            repo sync -j${NPROC_SYNC}
+            LTO=full BUILD_AOSP_KERNEL=1 ./build_lynx.sh
+            rsync -av --delete out/mixed/dist/ device/google/lynx-kernel/
+            ;;
+    esac
+}
+
+package_os () {
+    DEVICE=$1
+    # https://github.com/GrapheneOS/platform_development.git 
+    # Currently, the plan is to include your own made keys. This is here just in case we decide to do this on the fly which will definitely not be recommended at all.
+    # Packaging also signs the OS as well. 
+    if [[ $USE_GRAPHENEOS_TEST_KEYS == "true" ]]; then
+        git clone https://github.com/Reflux0301/grapheneos_test_keys.git /opt/build/grapheneos
```

Comment:
Why has this been done?

Reply:
Currently, `script/release.sh` is the current recommended way per the GrapheneOS documentation to package the OS into a zip file that is usable.

This shell script signs the APK's and APEX's first and then moves them into a zip file. Ideally, upstream would allow for `test-keys` to be used here or to have some mechanism to not sign and just zip it up. 

The only way that I know of currently to have a finalized zip file is to have keys available, to which, the user (or us) has to provide them, or we have to generate them on the fly. Doing this in the container isn't viable as this is a blocking operation where it asks the user for a password if they want to set one.

Reply:
Resolved in upcoming commit, symbolic links set to the AOSP keys